lib/streamlit/elements/map.py

```diff
@@ -64,19 +61,12 @@ def _get_zoom_level(distance):
 
     for i in range(len(ZOOM_LEVELS) - 1):
         if ZOOM_LEVELS[i + 1] < distance <= ZOOM_LEVELS[i]:
-            return i + 1
-
-
-def marshall(element, data, zoom=None):
-    """Marshall a proto with DeckGL chart info.
+            return i
```

Comment:
Why change to `i`?

Reply:
Based on this [documentation](https://github.com/uber/deck.gl/blob/master/docs/api-reference/map-view.md) the min zoom is 0 and the max is 20.

And with the +1 the view had a lot padding.